### PR TITLE
add first  version of sponsoring page

### PR DESCRIPTION
**Pull request by marcoemrich**
Please do proof reading, and changes as fit

### CODE REVIEW

File: src/sponsoring.ejs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                </li>
+                <li>Sponsors will be announced via the official JSCC <b>Twitter</b> account</li>
+            </ul>
+        </div>
+        <div class="col s12 m6">
+            <h5 class="header center">Why Sponsor?</h5>
+            <p class="block">
+                The JSCraftCamp is an Open Space conference that attracts diverse people from the JavaScript community
+                who really care about their craft. The crowd is self selected by investing time for their own application
+                (pull request).
```
Comment (davelosert) [inline src/sponsoring.ejs]: I find the sentence `The crowd is self selected by investing time for their own application`  at little bit confusing.
I guess the gross of what you want to say here is that the people attending the conference are there out of free will and curiosity, and are not forced by a company. And they show this by using their own time to put an application in.

What do you think about this:
"All Participants will be there on their private time and by there own drive and curiosity."

**Comment by davelosert** [comment]
@marcoemrich I really like the page and the layout. Great work!
I already took the liberty to fix some wordings and typos. I also left a comment on one sentence and made a proposal - would be great if you could review it. :smile:

**Comment by marcoemrich** [comment]
yes, it's hard to express in a short sentence. I wanted to explain, that because people apply by pr, they are really interested in the topic and not your typical 9to5-dev. I thinks term "self selection" is pretty established, but other than that it's rather difficult to express. May we just scratch the sentence. It's not that important, so less is more.

File: src/sponsoring.ejs (hunk 1 above)
Comment (marcoemrich) [inline src/sponsoring.ejs]: thanks for the fixes!

Comment (marcoemrich) [inline src/sponsoring.ejs]: oh, did you completely delete the 2nd part of the "Why Sponsor?" ?

Comment (marcoemrich) [inline src/sponsoring.ejs]: let's just scratch the confusing sentence :)

**Comment by wolframkriesing** [comment]
for some reason this changes the styling of list elements, see the main navigation (and the footer too)
bildschirmfoto 2017-05-01 um 23 52 50

**Comment by marcoemrich** [comment]
fixed it. thanks for finding the issue. I hadn't noticed.

**Pull request merged**